vcpkg/get_started/get-started.md

```diff
@@ -0,0 +1,168 @@
+
+    Now, let's break down what each line in the `CMakeLists.txt` file does:
+
+    - `cmake_minimum_required(VERSION 3.10)`: Specifies that the minimum version of CMake required to build the project is 3.10. If the version of CMake installed on your system is lower than this, an error will be generated.
+    - `project(HelloWorld)`: Sets the name of the project to "HelloWorld."
+    - `find_package(fmt CONFIG REQUIRED)`: Looks for the `fmt` library using its CMake configuration file. The `REQUIRED` keyword ensures that an error is generated if the package is not found.
+    - `add_executable(HelloWorld main.cpp)`: Adds an executable target named "HelloWorld," built from the source file `main.cpp`.
+    - `target_link_libraries(HelloWorld PRIVATE fmt::fmt)`: Specifies that the `HelloWorld` executable should link against the `fmt` library. The `PRIVATE` keyword indicates that `fmt` is only needed for building `HelloWorld` and should not propagate to other dependent projects.
+
+    Create the `main.cpp` file with the following content:
+
+    :::code language="cpp" source="../examples/snippets/get-started/main.cpp":::
+
+    In this `main.cpp` file, the `<fmt/core.h>` header is included for using the `fmt` library. The `main()` function then utilizes `fmt::print()` to output the "Hello World!" message to the console.
+
+## 4 - Build and run the project
+
+1. Run CMake configuration
+
+    To allow the CMake project system to recognize C++ libraries provided by vcpkg, you'll need to provide the `vcpkg.cmake` toolchain file. 
+
+    First, set the `VCPKG_ROOT` environment variable for the current shell session:
+
+    ::: zone pivot="os-linux,os-macos"
+
+    ```console
+    export VCPKG_ROOT=/path/to/your/vcpkg/directory
+    ```
+
+    > [!NOTE]
+    > Setting the `VCPKG_ROOT` using the `export` command will only affect the current shell session. To make this change permanent across sessions, you'll need to add the `export` command to your shell's profile script (e.g., `~/.bashrc` or `~/.zshrc`).
+
+    ::: zone-end
+
+    ::: zone pivot="os-windows"
+
+    ```console
+    set VCPKG_ROOT="/path/to/your/vcpkg/directory"
+    ```
+
+    > [!NOTE]
+    > Setting the `VCPKG_ROOT` environment variable using the `set` command will only affect the current shell session. To make this change permanent across sessions, you can use the `setx` command and restart the shell session.
+
+    ::: zone-end
+
+
+    Next, create a `CMakePresets.json` file in the "helloworld" directory with the following content:
+
+    :::code language="cmake" source="../examples/snippets/get-started/CMakePresets.json":::
+
+    This `CMakePresets.json` file contains a single "default" preset for CMake. It uses version 3 of the schema and sets the `CMAKE_TOOLCHAIN_FILE` variable, pointing to a toolchain file in the `VCPKG_ROOT` environment variable. This automates the process of specifying the toolchain when running CMake.
```

Comment:
```suggestion
    This `CMakePresets.json` file contains a single "default" preset for CMake. It sets the `CMAKE_TOOLCHAIN_FILE` variable, pointing to a toolchain file in the `VCPKG_ROOT` environment variable. This automates the process of specifying the toolchain when running CMake.
```

I think this could also use a hint more "why" over "what". It seems pretty self-evident that it will set `CMAKE_TOOLCHAIN_FILE` using `VCPKG_ROOT`, but I think it would help to call out that this is how CMake will consume libraries from vcpkg.